### PR TITLE
Derricks branch

### PR DESCRIPTION
**Pull request by Tactakon**
Headers and footers are set up on each necessary page with working icons that are wired to the corresponding pages. the logout icon will land you back to the landing page where you're able to log back in. the plus icon will bring you to the createplaylist page. and lastly, the home icon will bring you to the userplaylist page.

### CODE REVIEW

**Review by KinjalRele — COMMENTED** [review]
Can you put some descriptive comments on what you have done?
It becomes easier to understand what the code is doing.

File: templates/homeheader.html [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
-        <a class="navbar-brand" href="userPlaylistpage.html" style="color: #1D1D1D; font-size: 32px; font-family: Arial, Helvetica, sans-serif; font-weight: 800; padding-right: 30px;">VERGE</a> <!--linked to user homepage--> 
+        <a class="navbar-brand" href="{{ url_for('userPlaylistpage')}}" style="color: #1D1D1D; font-size: 32px; font-family: Arial, Helvetica, sans-serif; font-weight: 800; padding-right: 30px;">VERGE</a> <!--linked to user homepage--> 
         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
           <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarNav">
           <ul class="navbar-nav mr-auto">
             <li class="nav-item">
-              <a class="nav-link" href="userPlaylistpage.html" style="padding-right: 20px;"><!--Linked to user homepage-->
+              <a class="nav-link" href="{{ url_for('userPlaylistpage')}}" target="_top" style="padding-right: 20px;"><!--Linked to user homepage-->
```
Comment (Tactakon) [inline templates/homeheader.html:17]: made home button go to user playlist page

File: templates/homeheader.html [hunk 2]
```diff
@@ -34,7 +34,7 @@
               </a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="#"> <!--should be linked to create playlist page-->
+              <a class="nav-link" href="{{ url_for('createPlaylistPage')}}" target="_top"> <!--should be linked to create playlist page-->
```
Comment (Tactakon) [inline templates/homeheader.html:37]: made plus icon go to create playlist page

File: templates/homeheader.html [hunk 3]
```diff
@@ -67,8 +67,7 @@
               </a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="landingpage.html" style="font-size: large;" > <!--Should be linked to the landing page-->
-
+              <a class="nav-link" href="{{ url_for('main')}}" target="_top" style="font-size: large;" action="/homeheader" method="POST" > <!--Should be linked to the landing page-->
```
Comment (Tactakon) [inline templates/homeheader.html:70]: made signout icon take you back to the landing page

File: main.py [hunk 4]
```diff
@@ -112,16 +109,18 @@ def main():
...
-# landfooter.html
-@app.route('/footer')
-def footer():
-    return flask.render_template('landfooter.html')
-
-# homeheader.html
+#homeheader.html
 @app.route('/homeheader')
 def homeheader():
+    if request.method == 'POST':
```
Comment (Tactakon) [inline main.py:115]: it wont let me remove this in this PR for some reason but these two lines are no longer needed

**Comment by Tactakon** [comment]
most of this pr is making sure the footers and headers display properly and that the pages are wired to the homeheader properly

**Review by KinjalRele — APPROVED** [review]
Good Description!

**Pull request merged**